Gemfile

```diff
@@ -79,13 +79,15 @@ group :development, :test do
   gem 'pry-byebug'
   gem 'rspec-rails', '~> 3.5.2'
   gem 'slim_lint'
+  gem 'teaspoon-mocha'
   gem 'thin'
 end
 
 group :test do
   gem 'axe-matchers'
   gem 'capybara-screenshot', github: 'mattheworiordan/capybara-screenshot'
   gem 'codeclimate-test-reporter', require: false
+  gem 'coffee-script'
```

Comment:
out of curiosity, what is this used for? none of our tests or JS files are in `.coffee`?